### PR TITLE
EDU-1445 Redirect archived documents

### PR DESCRIPTION
**Pull request by adrianaluchian**
Closes https://educandu.atlassian.net/browse/EDU-1445

### CODE REVIEW

File: src/components/pages/document.yml [hunk 1]
```diff
@@ -80,6 +80,9 @@ document:
   archiveAlert:
     en: This document is archived and accessible only by users with admin or maintainer role.
     de: Dieses Dokument ist archiviert und nur für Benutzer mit der Rolle Admin oder Maintainer zugänglich.
+  archiveAlertRedirectMarkdown:
+    en: The other users are automatically redirected to [this document]({documentUrl}).
+    de: Die andere Benutzer sind zu [dieses Dokument]({documentUrl}) weitergeleitet.
```
Comment (ahelmberger) [inline src/components/pages/document.yml:85]: Die andere**n** Benutzer werden automatisch zu \[diese**m** Dokument]({documentUrl}) weitergeleitet.

**Comment by ahelmberger** [comment]
Really nice user experience with the message on top :heart:

File: src/components/pages/document.yml (hunk 1 above)
Comment (adrianaluchian) [reply src/components/pages/document.yml]: Thank you for the highlighting 😮‍💨 Corrected!

**Comment by adrianaluchian** [comment]
> Really nice user experience with the message on top ❤️

Thank you! ☺️

**Review by ahelmberger — APPROVED** [review]

**Pull request merged**